### PR TITLE
Automatically upload reference figures to the repo

### PR DESCRIPTION
**Pull request by Cadair**
Also a CI cleanup

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -92,5 +93,6 @@ jobs:
         - py36_test
         - py37_test
         - py38_test_online
+        - py38_test_32bit
         - py36_test_oldestdeps
```
Comment (nabobalis) [inline azure-pipelines.yml:97]: Do we need two 36 tests?

Comment (Cadair) [reply azure-pipelines.yml:97]: the other one is macos

Comment (Cadair) [reply azure-pipelines.yml:97]: You think one should be 3.7?

Comment (nabobalis) [reply azure-pipelines.yml:97]: Or remove the first 36 test?

Comment (Cadair) [reply azure-pipelines.yml:97]: that's the macos build.

Comment (nabobalis) [reply azure-pipelines.yml:97]: Run oldest deps on macos then?

**Pull request closed without merging**